templates/admin/fast-settings-footer.php

```diff
@@ -26,11 +28,19 @@
 				?>
 			</a>
 		</li>
+		<?php if ( empty( $fastwc_app_id ) ) : ?>
 		<li class="fast-footer-link">
-			<a href="<?php echo esc_url( $fastwc_setting_fast_onboarding_url ); ?>" target="_blank" rel="noopener" title="<?php esc_attr_e( 'Login to the Fast Seller Dashboard', 'fast' ); ?>">
+			<a href="<?php echo esc_url( $fastwc_setting_fast_onboarding_url ); ?>" target="_blank" rel="noopener" title="<?php esc_attr_e( 'Become a Seller', 'fast' ); ?>">
+				<?php esc_html_e( 'Become a Seller', 'fast' ); ?>
+			</a>
+		</li>
+		<?php endif; ?>
+		<li class="fast-footer-link">
+			<a href="<?php echo esc_url( $fastwc_setting_fast_dashboard_url ); ?>" target="_blank" rel="noopener" title="<?php esc_attr_e( 'Login to the Fast Seller Dashboard', 'fast' ); ?>">
 				<?php esc_html_e( 'Seller Login', 'fast' ); ?>
 			</a>
 		</li>
+		<?php endif; ?>
```

Comment:
@corywebb Is this required here? I see conditional if has an end on line 37.

Reply:
This is now fixed. Good catch!